packages/google_maps_flutter/google_maps_flutter_web/lib/src/marker_clustering.dart

```diff
@@ -0,0 +1,205 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+// ignore_for_file: public_member_api_docs, non_constant_identifier_names
```

Comment:
```suggestion
// ignore_for_file: non_constant_identifier_names
```

I guess `non_constant_identifier_names` is needed to keep the capitalization of things coming from the JS library?